example_configs/bootstrap/bootstrap.md

```diff
@@ -261,6 +261,9 @@ spec:
             - name: DO_CLEANUP
               value: "true"
 
+            - name: LLDAP_FORCE_SET_PASSWORD
```

Comment:
so, IIUC, this PR makes the default behavior be "Only set the password if the user doesn't exist" which makes the password field a "default password", and this flag enables forcing the setting.

What I don't understand is: why have a default password? If you're creating it for users, then you _shouldn't_ have a default password, they should reset it themselves before logging in for the first time.